.kitchen.yml

```diff
@@ -1,96 +1,76 @@
 driver:
   name: vagrant
+  network:
+    - ["forwarded_port", {guest: 80, host: 8080}]
 
 provisioner:
   name: chef_zero
+  require_chef_omnibus: <%= ENV['CHEF_VERSION'] || 'latest' %>
```

Comment:
The above works fine, been using it internally for ages and is working for this project too.